### PR TITLE
Ccl2 122 chat mock data

### PR DESCRIPTION
**Pull request by Ant-Shell**
**Summary**
- This PR adds mock data for `Chat` and `Inbox` functionality
  - The mock data will update frequently as functionality is figured out, so this is PR is an initial commit

**Updates**
- Update route for `path=/` redirect `to="/home"` instead of `to="/landing-page"` per discussion
- Add file `src/data/chatData.jsx`
  - This file adds data structures for testing purposes
  - There are a quite few scenarios that can be tested, however this data should capture at least a couple of them

**Testing Instructions**
- The redirect to `/home` has been verified locally before submitting this PR
- The mock data has not been tested - however, it will be updated and tested thoroughly in a separate PR

### CODE REVIEW

File: src/data/chatData.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
+// Mock Data for Chat and Inbox testing
```
Comment (mpa-mxiang) [inline src/data/chatData.jsx:1]: Nice file type change! Thank you for doing it!

**Review by mpa-mxiang — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Ant-Shell  , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested changes.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by katiyelae — COMMENTED** [review]
Hey Anthony! 

Thanks for putting this together! One thing I thought about, for the Chat Inbox data we will pass in, I think we should add in a message and date property for the "users" array so that when we go back into the Chat Inbox to see the existing messages you can see that information as well and then we don't have to hard code it into the component. I hope that makes sense

Other than that, it looks great!

**Comment by Ant-Shell** [comment]
> Hey Anthony!
> 
> Thanks for putting this together! One thing I thought about, for the Chat Inbox data we will pass in, I think we should add in a message and date property for the "users" array so that when we go back into the Chat Inbox to see the existing messages you can see that information as well and then we don't have to hard code it into the component. I hope that makes sense
> 
> Other than that, it looks great!

Thank you for the feedback - let's look into making this update when we work on mocking out the chat functionality.

**Pull request merged**